src/p2p/net_node.inl

```diff
@@ -578,50 +578,10 @@ namespace nodetool
     if(m_external_port)
       MDEBUG("External port defined as " << m_external_port);
 
-    // Add UPnP port mapping
-    if(m_no_igd == false) {
-      MDEBUG("Attempting to add IGD port mapping.");
-      int result;
-#if MINIUPNPC_API_VERSION > 13
-      // default according to miniupnpc.h
-      unsigned char ttl = 2;
-      UPNPDev* deviceList = upnpDiscover(1000, NULL, NULL, 0, 0, ttl, &result);
-#else
-      UPNPDev* deviceList = upnpDiscover(1000, NULL, NULL, 0, 0, &result);
-#endif
-      UPNPUrls urls;
-      IGDdatas igdData;
-      char lanAddress[64];
-      result = UPNP_GetValidIGD(deviceList, &urls, &igdData, lanAddress, sizeof lanAddress);
-      freeUPNPDevlist(deviceList);
-      if (result != 0) {
-        if (result == 1) {
-          std::ostringstream portString;
-          portString << m_listenning_port;
-
-          // Delete the port mapping before we create it, just in case we have dangling port mapping from the daemon not being shut down correctly
-          UPNP_DeletePortMapping(urls.controlURL, igdData.first.servicetype, portString.str().c_str(), "TCP", 0);
-
-          int portMappingResult;
-          portMappingResult = UPNP_AddPortMapping(urls.controlURL, igdData.first.servicetype, portString.str().c_str(), portString.str().c_str(), lanAddress, CRYPTONOTE_NAME, "TCP", 0, "0");
-          if (portMappingResult != 0) {
-            LOG_ERROR("UPNP_AddPortMapping failed, error: " << strupnperror(portMappingResult));
-          } else {
-            MLOG_GREEN(el::Level::Info, "Added IGD port mapping.");
-          }
-        } else if (result == 2) {
-          MWARNING("IGD was found but reported as not connected.");
-        } else if (result == 3) {
-          MWARNING("UPnP device was found but not recognized as IGD.");
-        } else {
-          MWARNING("UPNP_GetValidIGD returned an unknown result code.");
-        }
+    // add UPnP port mapping
+    if(!m_no_igd)
+      add_upnp_port_mapping(m_listenning_port);
```

Comment:
listenning not listening?

Reply:
Well, yea it's a spelling mistake but it has nothing to do with this PR. I can push a commit to fix it here though...
https://github.com/monero-project/monero/blob/fa73d6cbf1db12715bac5a8e5bbf1bffa74d0b59/src/p2p/net_node.h#L290